pom.xml

```diff
@@ -154,8 +154,8 @@
     <commons.math3.version>3.4.1</commons.math3.version>
     <!-- managed up from 3.2.1 for SPARK-11652 -->
     <commons.collections.version>3.2.2</commons.collections.version>
-    <scala.version>2.11.12</scala.version>
-    <scala.binary.version>2.11</scala.binary.version>
+    <scala.version>2.12.7</scala.version>
```

Comment:
The definition of the profile `scala-2.11` needs to change to set these back to 2.11 versions. The `scala-2.12` profile doesn't need to set these now, then. You can keep the enforcer rule in the 2.12 profile to ban 2.11 dependencies. Actually, the line banning 2.10 dependencies can be removed as that's already set in the main build's config.